ERCS/erc-xxxx.md

```diff
@@ -0,0 +1,201 @@
+---
+eip:
+title: Wallet Capabilities for Account Abstraction
+description: New Wallet Capabilities providing a way for dApps and Account Abstraction wallets to communicate
+author: Yoav Weiss (@yoavw), Alex Forshtat (@forshtat), Dror Tirosh (@drortirosh), Shahaf Nacson (@shahafn)
+discussions-to:
+status: Draft
+type: Standards Track
+category: ERC
+created:
+requires: 5792, 7702
+---
+
+## Abstract
+
+[EIP-5792](./eip-5792) defines a baseline JSON-RPC API for a communication between wallets and dapps.
+With EIP-5792, apps and wallets can communicate about any advanced features using "capabilities" - extensions
+to the base protocol that must be defined in separate documents.
+
+This proposal defines a set of "capabilities" the wallets may want to implement in order to provide a
+comprehensive support for Account Abstraction.
+
+## Motivation
+
+[ERC-4337](./eip-4337.md) introduced Account Abstraction and enabling Smart Contract Accounts to function as first-class citizens in Ethereum.
+However, while [ERC-4337](./eip-4337.md) and [ERC-7769](./eip-7769.md) defines a low-level RPC API for Account Abstraction,
+it does not specify a way for advanced AA-aware dApps to communicate their supported features and parameters to the advanced AA Wallet Applications.
+
+This ERC addresses the issue by defining a structured set of capabilities tailored for Account Abstraction dApps and Wallet Applications.
+
+It extends the [EIP-5792](./eip-5792) wallet capability model to encompass critical aspects of AA,
+ensuring dApps can seamlessly adapt to different AA Wallets without requiring custom solutions.
```

Comment:
Is it an extension, or 5792 capabilities proposals?

Reply:
Rephrased